interp/constrextern.ml

```diff
@@ -914,8 +914,11 @@ let rec extern inctx scopes vars r =
   | GProj (p, c) ->
     let pr = extern_reference ?loc Id.Set.empty (ConstRef (Projection.constant p)) in
     CProj (pr, sub_extern inctx scopes vars c)
+  | GInt i ->
+     CPrim(Numeral (Uint63.to_string i,true)) (* FIXME *)
```

Comment:
Could you make it clear what is supposed to be fixed in the comment? I don't know what this one is about, maybe I'll find out as I read on.

Reply:
This one is probably just a reminder for me to check what the boolean means. It won't stay.

Reply:
I believe the Boolean is the sign; true means non-negative; false means negative.  See
https://github.com/coq/coq/blob/master/interp/notation.ml#L926-L929